spec/unit/interface_spec.rb

```diff
@@ -180,7 +180,7 @@ def add_options_to(&block)
           display_global_options "modulepath"
         end
       end
-      face.display_global_options =~ ["environment"]
+      expect(face.display_global_options).to match(["environment"])
```

Comment:
Looks good, but maybe the commit message is for a different commit?

Reply:
No, this code was throwing the deprecation warning about `Object#=~`. Changing it to this removed the warning :-D